### ISSUE TITLE
Migrate just crashed sqlserver

### ISSUE THREAD

**Issue by anaelj** [prisma/prisma#14611]
Hi Prisma Team! Prisma Migrate just crashed. 

## Command

`db pull`

## Versions
      
| Name        | Version            |
|-------------|--------------------|
| Platform    | windows            | 
| Node        | v14.17.0           | 
| Prisma CLI  | 4.1.1              | 
| Engine      | 8d8414deb360336e4698a65aa45a1fbaf1ce13d8| 

## Error
```
Error: [libs\datamodel\connectors\dml\src\model.rs:494:29] Hi there! We've been seeing this error in our error reporting backend, but cannot reproduce it in our own tests. The problem is that we have a primary key in the model `AD_OCOPROMOTOR` that uses the column `NROREG` which we for some reason don't have in our internal representation. If you see this, could you please file an issue to https://github.com/prisma/prisma so we can discuss about fixing this. -- Your friendly prisma developers. 
```

**Comment by janpio** [comment]
Hi @anaelj, thanks for reporting this. What database are you using and which version? Could you introspect this database before successfully with Prisma? Can you share the SQL schema of that database with us? (We can accept them in private at schemas@prisma.io, NDA if needed as well.)

**Comment by anaelj** [comment]
This error occurs when i run instrospect

prisma : 4.1.1
@prisma/client : 4.1.1
Microsoft SQL Server 2014 - 12.0.4100.1 (X64) 
Standard Edition (64-bit) on Windows NT 6.3 <X64> (Build 9600: ) (Hypervisor)

Sorry, i don´t have permission to share the schema.

**Comment by tomhoule** [comment]
This should be fixed in Prisma 4.2, to be released tomorrow (https://github.com/prisma/prisma-engines/pull/3108)
